code/modules/mob/living/carbon/human/death.dm

```diff
@@ -131,4 +131,17 @@
 /mob/living/carbon/human/proc/vr_disconnect()
 	if(remote_network)
 		SSvirtualreality.remove_robot(src, remote_network)
-		remote_network = null
\ No newline at end of file
+		remote_network = null
+
+/mob/living/carbon/human/proc/drop_all_limbs(var/skip_qdel, var/droplimb_type = DROPLIMB_BLUNT)
+	for(var/thing in organs)
+		var/obj/item/organ/external/limb = thing
+		var/limb_can_amputate = (limb.limb_flags & ORGAN_CAN_AMPUTATE)
+		limb.limb_flags |= ORGAN_CAN_AMPUTATE
+		limb.droplimb(TRUE, droplimb_type, TRUE, TRUE)
+		if(!QDELETED(limb) && limb_can_amputate)
+			limb.limb_flags &= ~ORGAN_CAN_AMPUTATE
+	for(var/obj/O in contents)
```

Comment:
Might want to have a proc for this that removes organs/implants/etc in case the `droplimb()` loop missed them. On Neb we have an atom proc for getting a list of droppable stuff that prunes organs or robot components etc.